src/i18n.js

```diff
@@ -208,6 +208,31 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'id-ID': {
+    translation: {
+      title: 'kartu WiFi',
```

Comment:
Should Kartu be capitalized?

Reply:
Yes, it should be capitalized. I'll fix it tomorrow.